src/components/ec/cuda/ec_cuda.h

```diff
@@ -115,6 +97,7 @@ typedef struct ucc_ec_cuda_executor_task_ops {
 typedef struct ucc_ec_cuda_executor {
     ucc_ee_executor_t                super;
     ucc_ec_cuda_executor_mode_t      mode;
+    uint64_t                         requested_ops;
     ucc_ec_cuda_executor_task_ops_t  ops;
     ucc_spinlock_t                   tasks_lock;
     ucc_ec_cuda_executor_state_t     state;
```

Comment:
not related to this PR, but shouldn't `state` need to be allocated with `cudaHostAlloc`?

Reply:
it's allocated with cudaHostAlloc see ucc_ec_cuda_ee_executor_mpool_ops